### ISSUE TITLE
"Update media source" tootctl / S2S AP

### ISSUE THREAD

**Issue by shleeable** [mastodon/mastodon#22055]
### Pitch

A lot of us instance administrators are experiencing bill shock from bad choices (mostly using AWS).
![image](https://user-images.githubusercontent.com/17537000/205530006-2902cb5e-1185-4ec4-b9a7-06d073aff39c.png)

I need to be able to move my previous older media from AWS to wasabi (behind a media / s3 alias)

---

I desperately need some kind of S2S broadcast to change all of my toots media paths for all of my instances toots

`tootctl media migrate FROMURL TOURL`

and this should tell every instance in the fediverse to select all of my instances toots, and find/replace https://s3.ap-southeast-2.amazonaws.com/files.aus.social with https://mediacdn.aus.social/

from 
https://s3.ap-southeast-2.amazonaws.com/files.aus.social/media_attachments/files/000/000/042/original/770e76b97eab5419.png
to 
https://mediacdn.aus.social/media_attachments/files/000/000/042/original/770e76b97eab5419.png 

### Motivation

I want to decommission my AWS S3 bucket, but I need to leave it online for older media

**Comment by ineffyble** [comment]
So this is for the scenario where a server has deleted its cache of the media (e.g. with retention policy or tootctl cron) but kept the post, and then someone views the post, triggering the server to try and pull the media to cache again?

**Comment by shleeable** [comment]
Yeah, I assume the third party instance won't try to update the toot path.
